apple-pie/styles/style.css

```
@@ -0,0 +1,78 @@
+
+body{
+    color: grey;
+    margin: 0;
+    border: 0;
+    padding: 0;
+    margin-left: 400px;
+    margin-right: 400px;
+    font-weight: bold;
+    font-family: 'Roboto', sans-serif;
+
+}
+.applePieImage{
+    margin: auto;
+    display: flex;
+    justify-content: center;
+}
+
+.applePieImage img{
+    position: flex;
```

Aquí para poder utilizar flexbox hay que cambiar  a: 
```suggestion
    display: flex;
```